.github/actions/generate-help-docs.mjs

```diff
@@ -14,9 +15,16 @@ output = output.replace(
 const readmePath = join('..', '..', 'README.md')
 const readme = await readFile(readmePath, 'utf-8')
 
-const updatedReadme = readme.replace(
-  /(?<=<!-- GENERATED START -->\n\n```\n)[\s\S]*?(?=```\n\n<!-- GENERATED END -->)/,
-  output,
-)
+let updatedReadme = readme.replace(/(?<=<!-- USAGE START -->\n\n```\n)[\s\S]*?(?=```\n\n<!-- USAGE END -->)/, output)
+
+const codemods = TRANSFORM_OPTIONS.map(({ value, description, version }) => {
+  return `### ${value.replaceAll('-', ' ')} (v${version})
+
+${description}
+
+`
+}).join('')
+
+updatedReadme = updatedReadme.replace(/(?<=<!-- CODEMODS START -->\n\n)[\s\S]*?(?=<!-- CODEMODS END -->)/, codemods)
```

Comment:
Can you move `codemods` definition before `updateReadme` variable and chain `.replace` in a single run?

Reply:
done